extension/memorylimiterextension/README.md

```diff
@@ -0,0 +1,101 @@
+# Memory Limiter extension
+
+| Status                   |                   |
+|--------------------------|-------------------|
+| Stability                | [alpha]           |
+| Distributions            | [core], [contrib] |
```

Comment:
Once mdatagen is enabled, this should be replaced with `<!-- status autogenerated section -->`. `make gogenerate` will automatically fill this.

Reply:
done